homeassistant/components/netatmo/__init__.py

```diff
@@ -103,19 +106,20 @@ def setup(hass, config):
 
     if config[DOMAIN][CONF_WEBHOOKS]:
         webhook_id = hass.components.webhook.async_generate_id()
-        NETATMO_WEBHOOK_URL = hass.components.webhook.async_generate_url(
-            webhook_id)
+        hass.data[
+            DATA_WEBHOOK_URL] = hass.components.webhook.async_generate_url(
```

Comment:
There's actually nothing in these webhook callback decorated helpers that makes them necessary to be run in the event loop.

So technically we can just call them here directly, even though it's not according to our standards.

Otherwise I think we would have to refactor netatmo more and use a coroutine function for component setup.

Reply:
Uhm, refactoring to async might be out of scope for this PR. So if you say technically it's not a problem, then the changes I have made would address all the other issues you have commented on. 🤔

Reply:
Oh, fixed the remaining lint error.